RevenueCatUI/PaywallView.swift

```diff
@@ -179,6 +191,19 @@ public struct PaywallView: View {
         }
     }
 
+    @ToolbarContentBuilder
+    private var toolbar: some ToolbarContent {
+        ToolbarItemGroup(placement: .destructiveAction) {
+            if self.displayCloseButton {
+                Button {
+                    self.dismiss()
+                } label: {
+                    Image(systemName: "xmark")
```

Comment:
In Android I used our own `close` button vector image for this, but I think it should be ok (probably preferrable) to use the system one.

Reply:
Yeah this uses SFSymbols which is definitely preferred.